pandas/_libs/parsers.pyx

```diff
@@ -215,6 +215,13 @@ cdef extern from "parser/tokenizer.h":
         int64_t header_start        # header row start
         uint64_t header_end         # header row end
 
+        bint allow_leading_cols     # Boolean: 1: can infer index col, 0: no index col
+        bint skip_header_end        # Boolean: 1: Header=None,
+                                    # 0 Header is not None
+                                    # This is used because header_end is
+                                    # uint64_t so there is no valid NULL
+                                    # value (i.e. header_end == -1).
```

Comment:
Think this would be more readable if you just moved the longer comment about `skip_header_end` above that line

Reply:
Fixed. Thanks.